plugin/secrets_service_account_key.go

```diff
@@ -175,25 +175,17 @@ func (b *backend) secretKeyRevoke(ctx context.Context, req *logical.Request, d *
 
 func (b *backend) getSecretKey(ctx context.Context, s logical.Storage, rs *RoleSet, keyType, keyAlgorithm string, ttl int) (*logical.Response, error) {
 	cfg, err := getConfig(ctx, s)
-	if err != nil {
-		return nil, errwrap.Wrapf("could not read backend config: {{err}}", err)
-	}
-	if cfg == nil {
-		cfg = &config{}
+	if rs.AccountId == nil {
```

Comment:
We should still check for nil on `cfg` and do an `err` check before this.

Reply:
Added back in. Looks like a mess up during the rebase/merge.